research/src/components/header.js

```diff
@@ -27,19 +28,33 @@ const Header = ({ siteTitle, githubURL }) => (
         <Logo siteTitle={siteTitle} />
       </span>
 
-      <a href={githubURL} target="_blank" rel="noreferrer noopener" style={{ color: 'inherit' }}>
-        <FontAwesomeIcon style={{ marginRight: '0.2em' }} icon={faGithub} /> GitHub
-      </a>
+      <div className="header-nav">
```

Comment:
Oh! I just realized I forgot to add these links into the mobile nav! Lemme fix that.

Reply:
Added.